examples/bri-3/src/bri/identity/bpiSubjects/capabilities/createBpiSubject/createBpiSubject.command.ts

```diff
@@ -1,13 +1,38 @@
 export class CreateBpiSubjectCommand {
 
-  name: string;
-  description: string;
-  publicKey: string;
+  private id: string; // TODO: Add uuid after #491
```

Comment:
no need for the command to have an ID - this is NestJS CQRS model. Also no need to have these fields private, i was only refering to our own domain objects for both comments.

Reply:
done